sycl/source/detail/linux_pi.cpp

```diff
@@ -0,0 +1,11 @@
+#include <dlfcn.h>
+#include <string>
+
+void *loadOsLibrary(const std::string &PluginPath) {
+  // TODO: Check if the option RTLD_NOW is correct.
+  return dlopen(PluginPath.c_str(), RTLD_NOW);
```

Comment:
Wondering if it would be useful to also set `RTLD_DEEPBIND` to prevent the plugin from picking up symbols from the  runtime, e.g., to prevent LLVM symbol collisions, though the flag is not POSIX compatible?